src/components/movie-detail.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import '../movie.css';
+type Props = {
+  movie: {
+    id: string;
+    title: string;
+    release_date: string;
+    poster_path: string;
+    vote_average: number;
+    overview: string;
+  },
+  genres: Genres[]
+}
+type Genres = {
+    id: number;
+    name: string;
+  }
+
+class MovieDetail extends Component<Props> {
```

Comment:
This component will have piece of state that will have the movie info fetched from detail api